tests/integration/helpers.py

```diff
@@ -62,8 +62,15 @@ async def get_proxied_url(ops_test: OpsTest, app_name: str, unit_num: Optional[i
 
     logging.debug(f"Endpoints proxied by Traefik/0: {proxied_endpoints}")
 
+    return {p: p["url"] for p in proxied_endpoints}
```

Comment:
There's a [test failure](https://github.com/canonical/cos-lite-bundle/actions/runs/8602009198/job/23883369465) related to this code:
```
  File "/home/runner/work/cos-lite-bundle/cos-lite-bundle/tests/integration/helpers.py", line 65, in get_all_proxied_urls
    return {p: p["url"] for p in proxied_endpoints}
  File "/home/runner/work/cos-lite-bundle/cos-lite-bundle/tests/integration/helpers.py", line 65, in <dictcomp>
    return {p: p["url"] for p in proxied_endpoints}
TypeError: string indices must be integers
```

I guess the issue is `p["url"]` because `p` is of type `str`?
Perhaps:
```python
return {k: v["url"] for k, v in proxied_endpoints.items()}
```
Not sure. Would be great to have example outputs documented inline.

Reply:
BTW, feel free to introduce `sh.juju.run("show-proxied-endpoints")`.
https://pypi.org/project/sh/